app/bundles/MessengerBundle/Config/services.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return function (ContainerConfigurator $configurator) {
+    $services = $configurator->services()
+        ->defaults()
+        ->autowire()
+        ->autoconfigure();
+
+    $services->load('Mautic\\MessengerBundle\\', '../')
+        ->exclude('../{'.implode(',', [
+                'Config',
+                'Tests',
+            ]).'}');
+
+    $services->alias(\Symfony\Component\Messenger\Transport\TransportFactory::class, 'messenger.transport_factory');
```

Comment:
@escopecz the alias is needed for auto-wiring for https://github.com/mautic/mautic/blob/3bf4588888b092d0cbebdaa575f5969c9ab57568/app/bundles/MessengerBundle/Validator/DsnValidator.php#L16 otherwise the service is inlined and no longer available for auto-wiring.